packages/kbn-unified-data-table/src/components/data_table.scss

```diff
@@ -78,7 +81,7 @@
     border-left: 0;
     border-right: 0;
   }
-  .euiDataGridRowCell.euiDataGridRowCell--controlColumn[data-gridcell-column-id='colorIndicator'] .euiDataGridRowCell__content {
+  .euiDataGridRowCell.euiDataGridRowCell--controlColumn[data-gridcell-column-id='colorIndicator'] {
```

Comment:
We added row color indicators in https://github.com/elastic/kibana/pull/192756/files and in this PR they don't show up any more. Looks like it has something to do with the cell height change. Would it be possible to fix?

Reply:
Thank you so much for catching this Julia, I was overly aggressive in updating some CSS selectors 🤦 This should be fixed now! Please let me know if not and if your eagle eyes spot anything else!